### PR TITLE
Update ru.po

### PR DESCRIPTION
**Pull request by Alexmelman88**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -973,54 +991,48 @@ msgstr ""
...
+#~ msgstr ""
+#~ "Чтобы удалить файлы и директории, относящиеся к юниту «$(unit)», "
+#~ "необходимо пройти аутентификацию."
+
+#~ msgid ""
+#~ "Authentication is required to freeze or thaw the processes of '$(unit)' "
+#~ "unit."
+#~ msgstr ""
+#~ "Чтобы отправить сигнал заморозки или разморозки процессам юнита "
+#~ "«$(unit)», необходимо пройти аутентификацию."
```
Comment (keszybz) [inline po/ru.po]: Why remove those?

Comment (Alexmelman88) [reply po/ru.po]: I updated .po file with this .pot https://l10n.gnome.org/POT/systemd.main/systemd.main.pot

Comment (yuwata) [reply po/ru.po]: Not sure I can follow, but please untouch or update them, instead of commenting out them.

Comment (Alexmelman88) [reply po/ru.po]: fixed

File: po/ru.po [hunk 2]
```diff
@@ -990,49 +990,3 @@ msgid ""
...
-#~ msgstr ""
-#~ "Чтобы удалить файлы и директории, относящиеся к юниту «$(unit)», "
-#~ "необходимо пройти аутентификацию."
-
-#~ msgid ""
-#~ "Authentication is required to freeze or thaw the processes of '$(unit)' "
-#~ "unit."
-#~ msgstr ""
-#~ "Чтобы отправить сигнал заморозки или разморозки процессам юнита "
-#~ "«$(unit)», необходимо пройти аутентификацию."
```
Comment (yuwata) [inline po/ru.po]: No no, please do not drop them.
Please read https://github.com/systemd/systemd/blob/main/docs/TRANSLATORS.md.
Especially, use the following:
```
ninja -C build/ systemd-update-po
```

**Review by yuwata — CHANGES_REQUESTED** [review]

File: po/ru.po (hunk 2 above)
Comment (Alexmelman88) [reply po/ru.po]: > No no, please do not drop them. Please read https://github.com/systemd/systemd/blob/main/docs/TRANSLATORS.md. Especially, use the following:
> 
> ```
> ninja -C build/ systemd-update-po
> ```

I updated the .po file and these strings were out of date

Comment (yuwata) [reply po/ru.po]: > I updated the .po file and these strings were out of date

I do not observe that for ru.po with `ninja systemd-update-po`. Is your git repository uptodate??

Comment (Alexmelman88) [reply po/ru.po]: Isn't this .pot relevant? https://l10n.gnome.org/POT/systemd.main/systemd.main.pot
 I used it

Comment (yuwata) [reply po/ru.po]: Not sure. I guess this is outdated. At least, it is not managed by us.

Comment (Alexmelman88) [reply po/ru.po]: Okay, so this PR can be closed, because ru.po is actual

**Pull request closed without merging**